pkg/cmd/server/kubernetes/node/node_config_test.go

```diff
@@ -102,6 +100,7 @@ func TestKubeletDefaults(t *testing.T) {
 			RegistryPullQPS:                5.0,
 			ResolverConfig:                 kubetypes.ResolvConfDefault,
 			KubeletCgroups:                 "",
+			CgroupsPerQOS:                  true,
```

Comment:
@derekwaynecarr @sjenning can you please review the new node config in the 1.6 rebase here?

We are seeing this on the CI AWS machines when running hack/end-to-end-docker.sh (in contrast, it's fine on a Fedora 25 machine):
```
I0411 12:32:33.862520    1647 kubelet.go:1757] skipping pod synchronization - [Failed to start ContainerManager failed to initialise top level QOS containers: root container /kubepods doesn't exist]
```
Do we miss any other settings?